test/util/solo_test.dart

```diff
@@ -1,18 +0,0 @@
-// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
-// for details. All rights reserved. Use of this source code is governed by a
-// BSD-style license that can be found in the LICENSE file.
-
-library linter.test.util.solo_test;
-
-import 'dart:io';
-
-import 'package:path/path.dart' as p;
-
-import '../rule_test.dart';
-
-/// Solo test runner.  Handy for debugging.
-main([args]) {
-  var ruleName = args[0];
-  var dir = new Directory(ruleDir).absolute;
-  testRule(ruleName, new File(p.join(dir.path, '$ruleName.dart')), debug: true);
```

Comment:
Neat!

Last reservation: a nice feature of the current `solo_test` jig is that you can easily run it from a debug session in your IDE.  Is there a similar supported flow for `pub run test`?